src/index.html

```diff
@@ -5,8 +5,301 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700& display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="header__logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover">
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog catalog--wrap">
+        <div class="card catalog__card card--wrap" data-qa="card">
```

Comment:
```suggestion
        <div class="card catalog__card" data-qa="card">
```
Why do you need this modifier